src/components/toplayer/pages/toplayer-interactive.jsx

```diff
@@ -83,7 +83,7 @@ const Toplayers = () => {
         <TopLayer size={SIZE.MEDIUM} lead withBugbox>
           <ContentBox>
             <ContentBoxContent spacedBottom={SPACING_SIZE.LARGE}>
-              <TextBit color={TEXT_BIT_COLOR.ALT} type={HEADER_TYPE.H1}>
+              <TextBit color={TEXT_BIT_COLOR.BLUE_SECONDARY} type={HEADER_TYPE.H1}>
```

Comment:
I see what you did here :D

Reply:
ALT colors are part of the past,
ignore @aju and merge it, fast

Reply:
@aju upsi :D